src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -262,7 +262,7 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         /// </summary>
         [Parameter(ParameterSetName = "StandardMethod")]
         [Parameter(ParameterSetName = "StandardMethodNoProxy")]
-        public virtual WebRequestMethod Method { get; set; } = WebRequestMethod.Default;
+        public virtual WebRequestMethod Method { get; set; } = WebRequestMethod.Get;
```

Comment:
Since WebRequestMethod is public we cannot change it (remove the Default). So please keep old default value here and below in conditions.